deploy/operator/utils.sh

```diff
@@ -133,6 +133,25 @@ function wait_for_boolean_field() {
     return 1
 }
 
+function wait_for_resource() {
+    object="$1"
+    namespace="$2"
+    interval="${4:-10}"
```

Comment:
Very minor, but did you mean? 
```suggestion
    interval="${3:-10}"
```